google/cloud/aiplatform/initializer.py

```diff
@@ -259,15 +259,18 @@ def create_client(
             "google-cloud-aiplatform",
         ).version
         client_info = gapic_v1.client_info.ClientInfo(
-            gapic_version=gapic_version, user_agent=f"model-builder/{gapic_version}"
+            gapic_version=gapic_version,
+            user_agent=
+            f"{self.user_agent_product}/{gapic_version} ({self.user_agent_comments})"
```

Comment:
Would like @dizcology input here on adding the user_agent_comment suffix. 

Will there be any issue here with converting the user_agent header for the default case from `"model_builder/v1"` to `"model_builder/v1 ()"`?

Reply:
Since this is not immediately needed I will remove it for now, once we have a need to add additional metrics in comment field we can add this back.